docs/development/windows_support.md

```diff
@@ -197,19 +210,89 @@ cmake -B build -GNinja . -DTHEROCK_AMDGPU_FAMILIES=gfx110X-dgpu
 > instructs CMake to compile with `/Z7` or equivalent, which is supported by
 > ccache.
 
-Ensure that MSVC is used by looking for lines like these in the logs:
+> [!TIP]
+> Ensure that MSVC is used by looking for lines like these in the logs:
+>
+> ```text
+> -- The C compiler identification is MSVC 19.42.34436.0
+> -- The CXX compiler identification is MSVC 19.42.34436.0
+> ```
+>
+> If you see some other compiler there, refer to the MSVC setup instructions up
+> in [Important tool settings](#important-tool-settings).
+
+### CMake build usage
 
-```text
--- The C compiler identification is MSVC 19.42.34436.0
--- The CXX compiler identification is MSVC 19.42.34436.0
+```bash
+cmake --build build --target therock-dist
 ```
 
-### Build
+This will start building using MSVC. Once the amd-llvm subproject is built,
+subprojects like the ROCm math libraries will be compiled using `clang.exe` and
+other tools from the amd-llvm toolchain.
+
+When the build completes, you should have a build of ROCm / the HIP SDK
+in `build/dist/rocm/`. See the [Build Artifacts guide](./artifacts.md) for more
+information about the build outputs.
+
+#### Building PyTorch
+
+Once you have `build/dist/rocm`, you can follow the instructions to build
+PyTorch from source over at
+[external-builds/pytorch](../../external-builds/pytorch/README.md).
+
+### Run tests
+
+Test builds can be enabled with `-DBUILD_TESTING=ON` (the default).
+
+Some subproject tests have been validated on Windows, like rocPRIM:
+
+```bash
+ctest --test-dir build/math-libs/rocPRIM/dist/bin/rocprim --output-on-failure
+```
+
+### Build troubleshooting and known issues
+
+#### `No such file or directory: (long path to a source file)`
+
+Ensure that you have long path support enabled in git (see
+[Important tool settings](#important-tool-settings)), then re-run
 
 ```bash
-cmake --build build
+python ./build_tools/fetch_sources.py
```

Comment:
Why fetching sources again?

Reply:
If long path support is not enabled in git, cloning will (silently...) skip such files. Need to poke git again after updating the setting.

Reply:
Excuse the noise, I missed the comment above. Sure, makes sense that way!

Reply:
Expanded on the comments to clarify.